lib/classes/class-errors.php

```diff
@@ -165,6 +170,13 @@ public function admin_notices() {
         if( TGM_Plugin_Activation::get_instance()->is_tgmpa_page() ) {
           return;
         }
+
+        //enqueue dismiss js for ajax requests
+        $script_path = Utility::path( 'static/scripts/ud-dismiss.js', 'url' );
```

Comment:
Please, be sure, script is being loaded ONLY if any notice or warning message is shown.